frontend/src/views/omni/Modals/UserCreate.vue

```diff
@@ -40,25 +45,28 @@ import { canManageUsers } from "@/methods/auth";
 import TSelectList from "@/components/common/SelectList/TSelectList.vue";
 import TInput from "@/components/common/TInput/TInput.vue";
 import { AuthType, authType } from "@/methods";
+import TNotification from "@/components/common/Notification/TNotification.vue";
+
+const notification: Ref<Notification | null> = shallowRef(null);
 
 const identity = ref("");
 const router = useRouter();
 
 const roles = [RoleNone, RoleReader, RoleOperator, RoleAdmin]
 
-const role: Ref<string> = ref(RoleReader);
+const role: Ref<string> = ref(RoleReader)
 
 const handleUserCreate = async () => {
   if (identity.value === "") {
-    showError("Failed to Create User", "User email is not defined");
+    showError("Failed to Create User", "User email is not defined", notification);
```

Comment:
This was kinds of a UI bug, the error was dislpayed behind the Create User popup on the main screen. I reworked here a bit to move it into the popup.